hnsecure/src/test/java/ca/bc/gov/hlth/hnsecure/parsing/UtilTest.java

```diff
@@ -32,4 +32,26 @@ public void testGetSendingFacility_whenBlank() {
 		assertEquals(expectedValue, actualValue);
 	}
 	
+	@Test
+	public void testhexadecimallength_whenMgCntrlIdIsBlank() {
```

Comment:
A better name for this test method would be testConvertStringToHex_msgControlIdIsBlank or some to that effect. Method being tested first followed by test case/scenario. Same applies to other test methods.